src/libs/actions/Policy.js

```diff
@@ -718,7 +719,23 @@ function generateDefaultWorkspaceName(email, policyList) {
         defaultWorkspaceName = 'My Group Workspace';
     }
 
-    return hasPolicies ? `${defaultWorkspaceName} ${policyListLength + 1}` : defaultWorkspaceName;
+    if (defaultWorkspaceName === '') {
+        return defaultWorkspaceName;
+    }
+    console.log('this is all the polices ', getAllPolicies());
+
+    // Check if this name already exists in the policies
+    let suffix = 0;
+    _.forEach(getAllPolicies(), (policy) => {
+        // Get the name of the policy
+        const {name} = policy;
+
+        if (_.includes(name, defaultWorkspaceName)) {
+            suffix += 1;
+        }
```

Comment:
I think we just want to check if it is the same, includes may mean that

My group policy for fun 

and then another policy called

My group policy would return true making it 

My group policy 1